my-sms/spec/controllers/students_controller_spec.rb

```diff
@@ -23,119 +25,115 @@
 # removed from Rails core in Rails 5, but can be added back in via the
 # `rails-controller-testing` gem.
 
-RSpec.describe StudentsController, :type => :controller do
-
-  # This should return the minimal set of attributes required to create a valid
-  # Student. As you add validations to Student, be sure to
-  # adjust the attributes here as well.
-  let(:valid_attributes) {
-    skip("Add a hash of attributes valid for your model")
-  }
-
-  let(:invalid_attributes) {
-    skip("Add a hash of attributes invalid for your model")
-  }
-
-  # This should return the minimal set of values that should be in the session
-  # in order to pass any filters (e.g. authentication) defined in
-  # StudentsController. Be sure to keep this updated too.
-  let(:valid_session) { {} }
-
-  describe "GET #index" do
-    it "returns a success response" do
-      student = Student.create! valid_attributes
-      get :index, {}, valid_session
-      expect(response).to be_success
+RSpec.describe StudentsController, type: :controller do
+  let(:student) { FactoryBot.create(:student) }
+
+  describe 'GET #index' do
+    subject { get :index }
+    it 'returns a success response' do
+      should be_success
+      should render_template('index')
     end
   end
 
-  describe "GET #show" do
-    it "returns a success response" do
-      student = Student.create! valid_attributes
-      get :show, {:id => student.to_param}, valid_session
-      expect(response).to be_success
+  describe 'GET #show' do
+    subject { get :show, { id: student.to_param } }
+    it 'returns a success response' do
+      should be_success
+      should render_template('show')
```

Comment:
We usually look to keep to a single assertion per expectation/it block, but this is a good example of where it makes more sense to group both into the one test. Good work.